download-hex.js

```diff
@@ -0,0 +1,36 @@
+(function () {
+  var http = require('http')
```

Comment:
I like to use [`simple-get`](https://github.com/feross/simple-get) instead of raw http, since it can handle the 'data' event concatenation, the JSON, as well as the headers. Just FYI